modyn/config/schema/pipeline/evaluation.py

```diff
@@ -0,0 +1,390 @@
+"""
```

Comment:
Taken from `schema/pipeline.py` and adjusted the config according the the changes in evaluation logic.